tests/integration_cases/test_raft.rs

```diff
@@ -3742,8 +3753,11 @@ fn test_add_learner() {
     let mut n1 = new_test_raft(1, vec![1], 10, 1, new_storage());
     n1.add_learner(2);
 
-    assert_eq!(n1.prs().learner_nodes(), vec![2]);
-    assert!(n1.prs().learners()[&2].is_learner);
+    assert_eq!(
+        n1.prs().learner_ids().iter().cloned().collect::<Vec<_>>(),
```

Comment:
Ditto.

Reply:
It's the wrong data type. To make you happy I will unwrap the value and check it directly.